src/AcmActionGroup/AcmActionGroup.tsx

```diff
@@ -18,18 +18,20 @@ export function AcmActionGroup(props: { children: React.ReactNode[] }) {
 
     return (
         <Flex className={classes.group}>
-            {props.children.map((child, i) => {
-                if (i + 2 > props.children.length) {
-                    return <FlexItem key={i}>{child}</FlexItem>
-                } else {
-                    return (
-                        <Fragment key={i}>
-                            <FlexItem>{child}</FlexItem>
-                            <Divider isVertical />
-                        </Fragment>
-                    )
-                }
-            })}
+            {props.children
+                .filter((child) => !!child)
```

Comment:
So if the component returned null this works?

Reply:
Worth a shot, I think I had to remove the previous null check because the components were throwing an exception when we try to use `recoil` in the console repo.